### ISSUE TITLE
[Accessibility] [TalkBack] PanResponder callbacks do not fire when TouchableOpacity is pressed and TalkBack is on

### ISSUE THREAD

**Issue by bpander** [facebook/react-native#28638]
## Description
On Android with TalkBack on, TouchableOpacity components do not trigger PanResponder callbacks when pressed if the TouchableOpacity has on onPress prop and its focusable prop is not `false`.

## React Native version:
System:
    OS: macOS Mojave 10.14.6
    CPU: (8) x64 Intel(R) Core(TM) i7-7820HQ CPU @ 2.90GHz
    Memory: 89.05 MB / 16.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 10.2.1 - ~/.nvm/versions/node/v10.2.1/bin/node
    Yarn: 1.16.0 - /usr/local/bin/yarn
    npm: 5.6.0 - ~/.nvm/versions/node/v10.2.1/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  SDKs:
    iOS SDK:
      Platforms: iOS 13.2, DriverKit 19.0, macOS 10.15, tvOS 13.2, watchOS 6.1
  IDEs:
    Android Studio: 3.5 AI-191.8026.42.35.5977832
    Xcode: 11.3.1/11C504 - /usr/bin/xcodebuild
  npmPackages:
    react: 16.9.0 => 16.9.0 
    react-native: 0.61.5 => 0.61.5

## Steps To Reproduce
1. Create a PanResponder and apply its panHandlers to a View.
2. Add a TouchableOpacity inside the View but don't give it an onPress prop.
3. View the app in Android with TalkBack on.
4. Move the accessibility focus to the TouchableOpacity.
5. Double-tap to activate, observe the pan responder callbacks fire.
6. Add both an onPress prop and `focusable={false}` prop to the TouchableOpacity
7. Double-tap to activate, observe the pan responder callbacks fire.
8. Remove the `focusable={false}` prop.
9. Double-tap to activate, observe the pan responder callbacks DO NOT fire.
10. Double-tap-and-hold, observe the pan responder callbacks do fire.

Furthermore, double-tapping a non-TouchableOpacity element such as a Text component _will_ trigger PanResponder callbacks.

## Expected Results
The behavior of a TouchableOpacity should be consistent with itself, iOS, iOS with VoiceOver, and Android-with-TalkBack-off. Double-tapping a TouchableOpacity should trigger PanResponder callbacks (and arguably parents' onTouchStart callbacks too) regardless of whether or not the TouchableOpacity has an onPress or focusable prop.

## Snack, code example, screenshot, or link to a repository:
```
import React, { useState, useCallback } from 'react';
import { View, TouchableOpacity, Text, PanResponder } from 'react-native';

export default () => {

    const callback = useCallback(() => Boolean(console.log('fired')), []);

    const [panResponder] = useState(PanResponder.create({
        onMoveShouldSetPanResponderCapture: callback,
        onStartShouldSetPanResponderCapture: callback,
    }));

    return (
        <View
            style={{ justifyContent: 'center', alignItems: 'center', flex: 1 }}
            onTouchStart={callback}
            {...panResponder.panHandlers}
        >
            <TouchableOpacity onPress={() => console.log('press')}>
                <Text>Button</Text>
            </TouchableOpacity>
            <Text>Not a button</Text>
        </View>
    );
};
```
